src/db.rs

```diff
@@ -1649,6 +1649,48 @@ impl WriteBatch {
         }
     }
 
+    /// Remove database entries from start key to end key.
+    ///
+    /// Removes the database entries in the range ["begin_key", "end_key"), i.e.,
+    /// including "begin_key" and excluding "end_key". It is not an error if no
+    /// keys exist in the range ["begin_key", "end_key").
+    pub fn delete_range<K: AsRef<[u8]>>(&mut self, start_key: K, end_key: K) -> Result<(), Error> {
```

Comment:
What do you think to rename params like this ?
```suggestion
    pub fn delete_range<K: AsRef<[u8]>>(&mut self, from: K, to: K) -> Result<(), Error> {
```

Reply:
Go for it!